### PR TITLE
Add example manifests and a guide for local deployment

### PR DESCRIPTION
**Pull request by prmsrswt**
This PR adds a "Getting Started" guide for Observatorium. The guide is not super thorough but covers a lot of things users might have questions about.
- Running a local OIDC Provider (`hydra`) for authentication.
- Running the Observatorium stack in a kind cluster.
- Running `token-refresher` to proxy Prometheus remote write requests with proper auth.
- Running Prometheus outside the cluster to write data into Observatorium as a tenant.
- Running Grafana to query and visualize the data.

### CODE REVIEW

**Review by kakkoyun — COMMENTED** [review]
Wow! This looks amazing 💯 Thanks for this @prmsrswt 

Only thing I would love to see is having the Hydra configuration files checked in the repo. And utilize them with a tiny script. Besides to pedagogical purposes, it would be awesome to quickly start Observatorium locally and test things out. 

What do you think?

**Comment by prmsrswt** [comment]
Yes, that would be cool. I'll update the PR.

**Comment by prmsrswt** [comment]
@kakkoyun as you suggested I have added the config files and created a bash script `quickstart.sh`.

File: environments/local/configs/prom.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  scrape_interval: 15s
+
+scrape_configs:
+- job_name: prom
+  static_configs:
+  - targets:
+    - localhost:9090
+
+remote_write:
+- url: http://localhost:8080/api/metrics/v1/test-oidc/api/v1/receive
```
Comment (kakkoyun) [inline environments/local/configs/prom.yaml:11]:
```suggestion
- url: http://localhost:8080/api/metrics/v1/test-oidc/api/v1/receive

```

just a nit

File: environments/local/quickstart.sh [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  echo "- Cloning observatorium/token-refresher and building...  -"
+  echo "-------------------------------------------"
+  git clone https://github.com/observatorium/token-refresher tmp/token-refresher
+  cd tmp/token-refresher
+  make build
+  mv ./token-refresher ../bin/
+  cd -
+}
+
+prereq() {
```
Comment (kakkoyun) [inline environments/local/quickstart.sh:32]:
```suggestion
setup() {
```
Easier on the eyes.

File: environments/local/quickstart.sh [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+grafana() {
+  echo "-------------------------------------------"
+  echo "- Starting Grafana using docker...  -"
+  echo "-------------------------------------------"
+  (docker run -p 3000:3000 grafana/grafana:7.3.7 &> /dev/null) &
+  echo "Open http://localhost:3000 in your browser. Add Prometheus datasource with endpoint http://172.17.0.1:8080/api/metrics/v1/test-oidc."
+}
+
+deploy() {
+  hydra
```
Comment (kakkoyun) [inline environments/local/quickstart.sh:135]: I can understand the reason behind creating functions for each of them. However, I'd prefer to inline them if they are not used more than once.

**Review by kakkoyun — APPROVED** [review]
Amazing work! I have left a couple of non-blocker comments.

**Review by kakkoyun — APPROVED** [review]
Amazing 💯

**Pull request merged**